File: Day 3/if-else_statements.cpp
#include <iostream>

int main()
{
	int N;
	std::cin >> N;
	
	if (N % 2)
        std::cout << "Weird";
    else {
        if (N >= 6 && N <= 20)
            std::cout << "Weird";
        else std::cout << "Not Weird";
    }

    return 0;
}

File: Day 3/if-else_statements.hs
main = do
    x_temp <- getLine
    let x = read x_temp :: Int
    if not (mod x 2 == 0)
        then putStrLn "Weird"
        else if x >= 6 && x <= 20
            then putStrLn "Weird"
            else putStrLn "Not Weird"